lib/rspec/rails/autotest.rb

```diff
@@ -22,7 +22,7 @@
 
 $:.push(*Dir['vendor/rails/*/lib'])
 
-require 'active_support/core_ext'
+require 'active_support/all'
```

Comment:
```ruby
require 'active_support/deprecation'
require 'active_support/core_ext'
```
Results in: `uninitialized constant ActiveSupport::Autoload (NameError)`.
So, I then tried:
```ruby
require 'active_support/deprecation'
require 'active_support/autoload'
require 'active_support/core_ext'
```
Which resulted in: `Autotest style autotest/rails_rspec doesn't seem to exist. Aborting.`